src/utils.js

```diff
@@ -443,6 +348,7 @@ const api = {
 				.trim();
 		return run(GET_LK_REGISTRY_SCOPE) !== "undefined";
 	},
+	/* istanbul ignore next */
```

Comment:
Is this not being tested?

Reply:
nope, we decided not to test it  but for some reason the coverage was dumb once I updated. If you look at the file raw we have this comment about it:
```
// mocking console.log is not awesome, and this function only exists
		// for the purpose of avoiding having to mock console.log in tests,
		// so we're just going to go ahead and safely ignore this in coverage
```

Reply:
cool cool cool